index.html

```diff
@@ -0,0 +1,151 @@
+          <input
+            type="email"
+            id="email"
+            placeholder="Email"
+            required
+            class="input"
+            pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{3,}$"
+          />
+        </div>
+        <div class="pword">
+          <label for="password">password:</label>
+          <input
+            id="password"
+            type="password"
+            placeholder="Password"
+            required
+            class="input"
+          />
+        </div>
+        <div class="developer">
+          <label for="microverse">Microverse developers</label>
+          <select id="microverse">
+            <option id="student_name1">Gemechu</option>
+            <option id="student_name2">Ismail</option>
+            <option id="student_name3">Others</option>
+          </select>
+        </div>
+        <div class="recommend">
+          <div>
+            <label>Would you recommend Microverse to a friend?</label>
+          </div>
+          <div class="recommend-option">
+            <input
+              type="radio"
+              name="Recommend_Microverse"
+              value="definitely"
+              id="definitely"
+            />
+            <label for="definitely">Definitely</label>
+            <input
+              type="radio"
+              name="Recommend_Microverse"
+              value="MayBe"
+              id="maybe"
+            />
+            <label for="maybe">Maybe</label>
+            <input
+              type="radio"
+              name="Recommend_Microverse"
+              value="NotSure"
+              id="notsure"
+            />
+            <label for="notsure">Not sure</label>
+          </div>
+        </div>
+        <div class="skill">
+          <label>Microverse Technical skills</label>
+          <div>
+            <input
+              type="checkbox"
+              id="html&css"
+              name="language1"
+              value="HTML & CSS"
+            />
+            <label for="html&css"> HTML & CSS</label>
+          </div>
+          <div>
+            <input type="checkbox" id="ruby" name="language2" value="Ruby" />
+            <label for="ruby"> Ruby</label>
+          </div>
+          <div>
+            <input
+              type="checkbox"
+              id="database"
+              name="language3"
+              value="Database"
+            />
+            <label for="database"> Database</label>
+          </div>
+          <div>
+            <input
+              type="checkbox"
+              id="javascript"
+              name="language4"
+              value="Javascript"
+            />
+            <label for="javascript"> Javascript</label>
+          </div>
+          <div>
+            <input
+              type="checkbox"
+              id="algorithms"
+              name="language5"
+              value="Algorithms"
+            />
+            <label for="algorithms"> Algorithms</label>
+          </div>
+        </div>
```

Comment:
I think you should remove the div extra closing tag in line 36.

Reply:
Nice giving a polite suggestion about this line of code, but you can take it a step further by explaining to the author of the code why they should make some required changes. For example, _**"I think you should remove the div extra closing tag in line 36 because of ....."**_